gradle-module/src/main/java/org/example/Main.java

```diff
@@ -5,21 +5,23 @@ public class Main {
     public static final String CONSTANT = "constant";
 
     public static void main(String[] args) {
+        String s = "123123";
         System.out.println("Hello world!");
     }
 
     /*
 
     asda
-
-    asd
-    as
-    d
+sd
```

Comment:
[Severity: High] The test method throws a RuntimeException without any handling or context.
Consider adding exception handling or providing a meaningful message when throwing the exception.
Rationale: Unhandled exceptions can lead to application crashes and make debugging difficult. Providing context helps in understanding the error.
Reference: Java best practices suggest handling exceptions gracefully to improve application robustness.